### PR TITLE
Fullfunctionalwebsite

### PR DESCRIPTION
**Pull request by hakichris**
In this branch, we are building a full website based on SPA. 
- [x] create UI
- [x] use react router
- [x]  use css

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,29 @@
...
+          <div className="verticalLine" />
+          <li><Link to="/quote">Quote</Link></li>
+        </ul>
+      </nav>
+      <Routes>
+        <Route path="/home" element={<Home />} />
+        <Route path="/calculator" element={<Calculator />} />
+        <Route path="/quote" element={<Quote />} />
+      </Routes>
+    </>
```
Comment (thecodechaser) [inline src/App.js]: - [ ] The home component should display on the root route `/` to match the wireframe and it is supposed to display on the root route. If users open the app they have to click on the home button to go to the home page but it should open on the root route(home page) by default. Please fix this issue.
  - You can consider the below code.
```ruby
<li><Link to="/">Home</Link></li>
.........
.........
<Route path="/" element={<Home />} />

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @hakichris👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Nice UI design. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (hakichris) [reply src/App.js]: solved, thank you.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @hakichris👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Nice UI design. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**